content/kubermatic/master/installation/install_kubermatic_ee/_index.en.md

```diff
@@ -8,7 +8,7 @@ enterprise = true
 This chapter explains the installation procedure of Kubermatic into a pre-existing Kubernetes cluster.
 
 {{% notice note %}}
-At the moment you need to be invited to get access to Kubermatic's Docker registry before you can try it out. Please [contact sales](mailto:sales@loodse.com) to receive your credentials.
+At the moment you need to be invited to get access to Kubermatic's Docker registry before you can try it out. Please [contact sales](mailto:sales@kubermatic.com) to receive your credentials.
```

Comment:
@scheeles do we really have this email?

Reply:
yes all @loodse.com have and alias to @kubermatic.com